modules/validation/helpers.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2018 The Gitea Authors. All rights reserved.
```

Comment:
I am quite certain it is fine not to update this year... Maybe @techknowlogick knows better?

Reply:
We should probably think about our copyright header update policy, but as of right now the date is of when the first change to the file is made.

Reply:
ah OK, reverted that thx